util/crc32c.cc

```diff
@@ -571,7 +571,8 @@ std::string IsFastCrc32Supported() {
  * This version is from the folly library, created by Dave Watson <davejwatson@fb.com>
  *
 */
-#if defined HAVE_SSE42 && defined HAVE_PCLMUL
+#if (defined HAVE_SSE42 && defined HAVE_PCLMUL) || \
+    (defined ROCKSDB_PORTABLE && defined _MSC_VER)
```

Comment:
I thought you changed the build so that HAVE_SSE42 and HAVE_PCLMUL would be set on MSVC regardless of PORTABLE, so am not sure why we need this.

Reply:
Fixed